esp8266/coffee-button/coffee-button.ino

```diff
@@ -0,0 +1,142 @@
+#include <OneButton.h>
+#include <ESP8266WiFi.h> 
+#include <ESP8266HTTPClient.h>
+#include <WiFiClientSecure.h>
+
+// edit the SSID & password to match your local network
+const char* ssid = "<insert_ssid_here>"; 
+const char* password = "<insert_pwd_here>"; 
+const char* host = "hooks.zapier.com"; // edit the host adress, ip address etc. 
+String url = "<insert_hook_url>"; // edit the hook url
+const int httpsPort = 443;
+// SHA1 fingerprint of the certificate
+// if you use another provider, be sure to change the fingerprint
+const char fingerprint[] PROGMEM = "AF 21 4A 6C 2C E4 CE 6E 99 7B B8 EA 58 CF 57 6B C2 35 A4 0D";
+int adcvalue=0;
+int value = 0;
+int ledOff = LOW;
+int ledOn = HIGH;
+int greenLed = 2;
+int redLed = 13;
+
+// OneButton button(D6, true);
+
+void setup() {
+  // Initializing LEDs
+  pinMode(greenLed, OUTPUT);
+  pinMode(redLed, OUTPUT);
+  digitalWrite(greenLed, ledOff);
+  digitalWrite(redLed, ledOff);
+
+  // button.attachClick(handleclick);
+  
+  // Initializing Wifi-connection
+  Serial.begin(115200); 
+  delay(10); // We start by connecting to a WiFi network 
+  Serial.println(); 
+  Serial.println(); Serial.print("Connecting to "); 
```

Comment:
Maybe two \n instead of two Serial.println()

Reply:
Yep, haven't seen that one. Not even needed, TBH...